app/services/av_characterizer_service.rb

```diff
@@ -128,10 +128,12 @@ def extract_part(track, part_type)
 
   # @param [String] time a date with format like 'UTC 2020-02-27 06:06:04' - although sometimes UTC doesn't come through
   def to_iso_time(time)
+    time.gsub!(/[:|-]/, '') # strips : or - in date and time parts to handle cases where the date has colons or dashes
```

Comment:
It looks like the colon use case is well tested in this branch. Do you feel the hyphen case is also well tested or should we add a test or two?

Reply:
Like where everything is a hyphen and not mixed hyphen/colon from HB exception?  At the moment, it currently tests the original case and the new case represented in the HB alert: https://app.honeybadger.io/projects/68956/faults/62761741

Reply:
OK, sounds good to me, @peetucket

Reply:
The `|` is not needed here.